lib/xboxrt/libc_extensions/locale_ext_.h

```diff
@@ -0,0 +1,26 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// We don't really have locale support.
+// These are only stubs, providing just the bare minimum for libc++.
+
+typedef void *_locale_t;
+
+static _locale_t _create_locale (int category, const char *locale)
+{
+    return NULL;
+}
+
+static void _free_locale (_locale_t locale) {}
+
+#define _ENABLE_PER_THREAD_LOCALE 1
+#define _DISABLE_PER_THREAD_LOCALE 2
+static int _configthreadlocale (int per_thread_locale_type)
+{
+    return _DISABLE_PER_THREAD_LOCALE;
```

Comment:
MSDN says `-1` on failure (not being able to change it should be considered a failure to me). Would that work for libc++?

Reply:
I checked, and libc++ uses the return value as a parameter to this same function later on, but apart from that, there's no checking or other use of it. As long as we ignore the parameter completely, returning `-1` seems to be fine, so I changed it.